src/Pages/LedSign/LedSign.js

```diff
@@ -87,13 +89,22 @@ function LedSign(props) {
       );
     }
   }
-
+  
   useEffect(() => {
     async function checkSignHealth() {
       setLoading(true);
       const status = await healthCheck(props.user.firstName);
       if (status && !status.error) {
         setSignHealthy(true);
+        if(status.responseData && status.responseData.text) {
```

Comment:
Is this statement necessary? Because isn't there always going to be responseData?

Reply:
Yeah, it's to deal with the first time the sign gets response data from the led server. If the server gives blank information this if statement is there to ignore it and use the initial states of the hooks.